tools/CoherenceBuild/CoherenceBuild.cs

```diff
@@ -152,5 +155,62 @@ private static string FindLatest(string repoDirectory)
                 .FirstOrDefault();
         }
 
+        private void GenerateDependenciesFile()
+        {
+            var project = new XElement("Project");
+            var propertyGroup = new XElement("PropertyGroup");
+            project.Add(propertyGroup);
+
+            var universeCoherence = _reposToProcess.First(repo => string.Equals(repo.Name, "UniverseCoherence", StringComparison.OrdinalIgnoreCase));
+            var buildDirectory = GetBuildDirectory(universeCoherence);
+            foreach (var nugetPackageFile in Directory.GetFiles(buildDirectory, "microsoft.aspnetcore.mvc.core*.nupkg"))
+            {
+                if (!nugetPackageFile.EndsWith(".symbols.nupkg", StringComparison.OrdinalIgnoreCase))
+                {
+                    var packageInfo = GetPackageIdAndVersion(nugetPackageFile);
+                    var element = new XElement("AspNetCoreVersion", packageInfo.Item2);
+                    propertyGroup.Add(element);
+                    break;
+                }
+            }
+
+            var coreClr = _reposToProcess.First(repo => string.Equals(repo.Name, "CoreCLR", StringComparison.OrdinalIgnoreCase));
+            buildDirectory = GetBuildDirectory(coreClr);
+            foreach (var nugetPackageFile in Directory.GetFiles(buildDirectory, "*.nupkg"))
+            {
+                if (!nugetPackageFile.EndsWith(".symbols.nupkg", StringComparison.OrdinalIgnoreCase))
+                {
+                    var packageInfo = GetPackageIdAndVersion(nugetPackageFile);
+                    var element = new XElement(packageInfo.Item1, packageInfo.Item2);
```

Comment:
This would be generated like the following...here the element name is just the id of the package. Sure it makes the element name long but not sure i like dealing with custom element names like 'MicrosoftBuildVersion', 'DependencyModelVersion'...I felt if we follow the property names with their package names, it would just be easier...thoughts?
```xml
<Microsoft.Build>15.3.0-preview-000360-03</Microsoft.Build>
    <Microsoft.Build.Framework>15.3.0-preview-000360-03</Microsoft.Build.Framework>
    <Microsoft.Build.Runtime>15.3.0-preview-000360-03</Microsoft.Build.Runtime>
    <Microsoft.Build.Tasks.Core>15.3.0-preview-000360-03</Microsoft.Build.Tasks.Core>
    <Microsoft.Build.Utilities.Core>15.3.0-preview-000360-03</Microsoft.Build.Utilities.Core>
    <Microsoft.CSharp>4.4.0-preview2-25331-02</Microsoft.CSharp>
    <Microsoft.DotNet.PlatformAbstractions>2.0.0-preview2-002062</Microsoft.DotNet.PlatformAbstractions>
    <Microsoft.Extensions.DependencyModel>2.0.0-preview2-25331-01</Microsoft.Extensions.DependencyModel>
    <Microsoft.NET.Test.Sdk>15.3.0-preview-20170531-01</Microsoft.NET.Test.Sdk>
    <Microsoft.NETCore.App>2.0.0-preview2-25331-01</Microsoft.NETCore.App>
```